service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NotifyGatekeeperTest.java

```diff
@@ -49,16 +53,26 @@
     @Autowired
     private ObjectMapper mapper;
 
+    @Test
+    void shouldReturnErrorsWhenFamilymanNumberIsNotProvided() throws Exception {
```

Comment:
Can we caps 'FamilyMan' to keep consistent?

Reply:
C6 and C21 use 'Familyman' so we'll leave it like this for now to be consistent with them, as I don't want to change those files in this PR